crates/ironrdp-connector/src/lib.rs

```diff
@@ -97,6 +99,36 @@ pub struct Config {
     pub no_server_pointer: bool,
     /// If true, the INFO_AUTOLOGON flag is set in the [`ironrdp_pdu::rdp::ClientInfoPdu`].
     pub autologon: bool,
+    // for kerberos authentication
+    pub sspi_config: Option<SspiConfig>,
```

Comment:
This should not be part of the "connector" configuration. Instead, you can pass it to the `CredsspSquence::new` function. Also, this should probably be named `KerberosConfig` since it is exclusively used to build an sspi-rs `KerberosConfig`.
```rust
impl CredSspSequence {
    /* snip */

    pub fn new(
        connector: &ClientConnector,
        kerberos_config: Option<KerberosConfig>,
        server_name: ServerName,
        server_public_key: Vec<u8>,
    ) -> ConnectorResult<Self> {
        /* snip */
    }
}
```